app/models/viewmodels/CasesTabViewModel.scala

```diff
@@ -135,14 +110,12 @@ object CasesTabViewModel {
       CasesTab.tta(),
       CasesTab.ttb(),
       CasesTab.ttc()
-
     )
   )
 
-
   //TODO remove dummy correspondence example when writing queries for the tabs
   val corrApplicationExample: CorrespondenceApplication = CorrespondenceApplication(
-   Some("Starter"),
+    Some("Starter"),
     Some("Agent 007"),
     Address("New building", "Old Town", None, None),
     Contact("a name", "anemail@some.com", None),
```

Comment:
Could we remove these examples as well now?

Reply:
👍